code/modules/mob/living/living_health_procs.dm

```diff
@@ -191,6 +191,20 @@
 		return
 	remove_movespeed_modifier(MOVESPEED_ID_DROWSINESS)
 
+///Adjusts the blood volume, with respect to the minimum and maximum values
+/mob/living/proc/adjust_blood_volume(amount)
+	if(!amount)
+		CRASH("adjust_blood_volume called with [isnull(amount) ? "null" : "[amount]"] amount of blood.")
```

Comment:
Do you really want a crash out if adjust(0) is called?

Reply:
It really shouldn't happen shrug

Reply:
Ill change it to just if its null when Im home